changelog.txt

```diff
@@ -5,6 +5,17 @@ Changes
 The **signac-flow** package follows `semantic versioning <https://semver.org/>`_.
 The numbers in brackets denote the related GitHub issue and/or pull request.
 
+next
```

Comment:
Do we normally have two `next` fields in the change log?

Reply:
This is a weird thing about rST. It assumes the first heading style it sees is top level, second style is second level, etc. If we only use one heading block, it ends up flipping the heading sizes on the others. It is not a big deal but is minorly annoying if you only use one.